python/ray/serve/_private/request_context.py

```diff
@@ -0,0 +1,24 @@
+import contextvars
+from dataclasses import dataclass
+
+# Serve request context var which is used for storing the internal
+# request context information.
+# route_prefix: http url route path, e.g. http://127.0.0.1:/app
+#     the route is "/app". When you send requests by handle,
+#     the route is "N/A.
+# request_id: the request id is generated from http proxy, the value
+#     shouldn't be changed when the variable is set.
+# note:
+#   The request context is readonly to avoid potential
+#       async task conflicts when using it concurrently.
+
+
+@dataclass(frozen=True)
+class RequestContext:
+    route: str = "N/A"
+    request_id: str = "N/A"
+
+
+_serve_request_context = contextvars.ContextVar(
```

Comment:
Could we move everything in this file to the existing [`context.py`](https://github.com/ray-project/ray/blob/e5a8b1dd55817305ac85237cd99d8b6ed23df294/python/ray/serve/context.py) file? Is there a benefit in keeping it separate?

Reply:
done